sql/updates/world/3.3.5/9999_99_99_99_world_burst.sql

```diff
@@ -0,0 +1,9 @@
+UPDATE `spell_script_names` SET `ScriptName`="spell_q13264_q13276_q13288_q13289_burst_at_the_seams" WHERE `spell_id` = 59576;
+
+DELETE FROM `conditions` WHERE `SourceEntry` = 59576 AND `SourceTypeOrReferenceId` = 13;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(13, 2, 59576, 0, 0, 31, 0, 3, 31142, 0, 0, 0, 0, "", "Spell 'Burst at the Seams' targets Icy Ghoul"),
```

Comment:
```diff
-(13, 2, 59576, 0, 0, 31, 0, 3, 31142, 0, 0, 0, 0, "", "Spell 'Burst at the Seams' targets Icy Ghoul"),
-(13, 2, 59576, 0, 1, 31, 0, 3, 31147, 0, 0, 0, 0, "", "Spell 'Burst at the Seams' targets Vicious Geist"),
-(13, 2, 59576, 0, 2, 31, 0, 3, 31205, 0, 0, 0, 0, "", "Spell 'Burst at the Seams' targets Risen Alliance Soldier"),
-(13, 2, 59576, 0, 3, 31, 0, 4,     0, 0, 0, 0, 0, "", "Spell 'Burst at the Seams' targets Players");
+(13, 2, 59576, 0, 0, 31, 0, 3, 31142, 0, 0, 0, 0, '', 'Spell \'Burst at the Seams\' targets Icy Ghoul'),
+(13, 2, 59576, 0, 1, 31, 0, 3, 31147, 0, 0, 0, 0, '', 'Spell \'Burst at the Seams\' targets Vicious Geist'),
+(13, 2, 59576, 0, 2, 31, 0, 3, 31205, 0, 0, 0, 0, '', 'Spell \'Burst at the Seams\' targets Risen Alliance Soldier'),
+(13, 2, 59576, 0, 3, 31, 0, 4,     0, 0, 0, 0, 0, '', 'Spell \'Burst at the Seams\' targets Players');
```

Reply:
No need to change the double quotation marks into single quotation marks + escape backslash
(unless there is a codestyle requirement I have overlooked in the TC SQL codestyle guidelines).

Reply:
I think should be single following TC codestyle, but I'm not sure xD

Reply:
Are you sure this must effect also players?

Reply:
https://www.wowhead.com/quest=13289/thats-abominable#comments:id=492401
https://www.wowhead.com/quest=13289/thats-abominable#comments:id=536986
https://www.wowhead.com/quest=13289/thats-abominable#comments:id=629542
https://www.wowhead.com/quest=13289/thats-abominable#comments:id=1096777
https://www.wowhead.com/quest=13264/thats-abominable#comments:id=492404
https://www.wowhead.com/quest=13276/thats-abominable#comments:id=516353
https://www.wowhead.com/quest=13276/thats-abominable#comments:id=999289
https://youtu.be/mhvrLmicm4s?t=24s